### PR TITLE
Adds `world.cpu`

### LINKED ISSUE #78: `world.cpu` is not implemented
http://www.byond.com/docs/ref/#/world/var/cpu

### PR DESCRIPTION
**Pull request by ike709**
Not sure if this is the best way to handle it, but it seems to work.

Fixes #78

### CODE REVIEW

File: DMCompiler/DMStandard/Types/World.dm [hunk 1]
```diff
@@ -12,6 +12,7 @@
 	var/timeofday
 	var/realtime
 	var/tick_lag = 1
+	var/cpu = 100
```
Comment (ike709) [inline DMCompiler/DMStandard/Types/World.dm:15]: Didn't seem like the best idea to leave it null on the first tick, so instead lets presume the first tick is costly.

Comment (wixoaGit) [reply DMCompiler/DMStandard/Types/World.dm:15]: `world.cpu` starts off as a 0

File: OpenDreamRuntime/DreamRuntime.cs [hunk 2]
```diff
@@ -157,6 +157,7 @@ public class DreamRuntime
                 int elapsedTime = (int)(new DateTimeOffset(DateTime.Now).ToUnixTimeMilliseconds() - TickStartTime);
                 int tickLength = (int)(100 * WorldInstance.GetVariable("tick_lag").GetValueAsFloat());
                 int timeToSleep = tickLength - elapsedTime;
+                WorldInstance.SetVariable("cpu", new DreamValue(100 - timeToSleep));
```
Comment (wixoaGit) [inline OpenDreamRuntime/DreamRuntime.cs:160]: This assumes a `world.tick_lag` of 1. `world.cpu` should be a ratio of `elapsedTime` and `tickLength`.
```suggestion
                WorldInstance.SetVariable("cpu", new DreamValue((float)elapsedTime / tickLength * 100));
```

File: TestGame/code.dm [hunk 3]
```diff
@@ -49,6 +49,9 @@
 	    for(var/item in out3)
 	    	usr << "The output should be between 1 and 10: [item]"
 
+	verb/show_cpu()
+		usr << "CPU: [world.cpu]"
```
Comment (wixoaGit) [inline TestGame/code.dm:53]: Could this be moved to the `Stat` proc?

**Review by wixoaGit — CHANGES_REQUESTED** [review]

File: DMCompiler/DMStandard/Types/World.dm (hunk 1 above)
Comment (ike709) [reply DMCompiler/DMStandard/Types/World.dm:15]:
```suggestion
	var/cpu = 0
```

**Comment by ike709** [comment]
Done.

**Review by wixoaGit — APPROVED** [review]

**Pull request merged**